dao/src/main/java/com/epam/esm/hibernate/OrderRepository.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.esm.hibernate;
+
+import com.epam.esm.Repository;
+import com.epam.esm.exception.DaoException;
+import com.epam.esm.persistence.OrderEntity;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+public interface OrderRepository extends Repository<OrderEntity> {
+
+    List<OrderEntity> findOrderOfSpecificUser(long idUser, long idOrder);
+
+    @Transactional
+    int create(OrderEntity newOrder);
```

Comment:
Just interested: on retrieving the order you are using long idOrder as parameter but on creating returning int not long. It should be consistent.